### PR TITLE
Polyphemus batch cat download notifications

### PR DESCRIPTION
**Pull request by coleshaw**
This PR pulls out the ingest Slack notifications to separate ETLs, so that we can batch the ingested files together into a digest and run at separate times (i.e. once a day).

### CODE REVIEW

File: etna/lib/etna/clients/metis/workflows/metis_upload_workflow.rb [hunk 1]
```diff
@@ -110,8 +110,6 @@ class Upload
 
           def initialize(source_file: nil, next_blob_size: nil, current_byte_position: nil)
             self.source_file = source_file
-            self.next_blob_size = next_blob_size
```
Comment (corps) [inline etna/lib/etna/clients/metis/workflows/metis_upload_workflow.rb:113]: is there a reason for this deletion?

File: polyphemus/lib/etls/slack_notification_module.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require_relative "../helpers"
+
+module Polyphemus::WithSlackNotification
+  include WithLogger
+
+  def slack(title, channel, msg)
```
Comment (corps) [inline polyphemus/lib/etls/slack_notification_module.rb:6]: Interestingly, I implemented something similar, too, so I'll fold in my PR with yours after.

File: etna/lib/etna/clients/metis/workflows/metis_upload_workflow.rb (hunk 1 above)
Comment (coleshaw) [reply etna/lib/etna/clients/metis/workflows/metis_upload_workflow.rb:113]: It seems redundant with the subsequent two lines of code?

Comment (corps) [reply etna/lib/etna/clients/metis/workflows/metis_upload_workflow.rb:113]: oh, hmmm, I see.

**Review by corps — APPROVED** [review]

**Pull request merged**